__init__.py

```diff
@@ -0,0 +1,44 @@
+# this module instantiates the app and ties Flask blueprints separated by functionality
+
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import LoginManager
+
+
+db = SQLAlchemy()
+
+
+def create_app():
+    app = Flask(__name__)
+
+    app.config['SECRET_KEY'] = 'cowsgomoo'
```

Comment:
can you please keep the secret key in a different file like the third option in the screenshot attached below? 

![image](https://user-images.githubusercontent.com/61632192/216053646-25d22b83-974d-45d4-b3d1-5bd0892da0b5.png)

Reply:
I've changed the code to read a secret key string from our .env file.